Sources/Hummingbird/Codable/JSON/JSONCoding.swift

```diff
@@ -12,18 +12,20 @@
 //
 //===----------------------------------------------------------------------===//
 
+import NIOFoundationCompat
+
 import struct Foundation.Date
 import class Foundation.JSONDecoder
 import class Foundation.JSONEncoder
-import NIOFoundationCompat
 
 extension JSONEncoder: ResponseEncoder {
     /// Extend JSONEncoder to support encoding `Response`'s. Sets body and header values
     /// - Parameters:
     ///   - value: Value to encode
     ///   - request: Request used to generate response
     ///   - context: Request context
-    public func encode(_ value: some Encodable, from request: Request, context: some RequestContext) throws -> Response {
+    public func encode(_ value: some Encodable, from request: Request, context: some RequestContext) throws -> Response
+    {
```

Comment:
This type of formatting is super annoying, is that configurable?

Reply:
Yeah I haven't worked out how to stop that. Not very keen on it